client-go/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2014 Engine Yard, Inc.
```

Comment:
I'm not a lawyer, but my arguments for leaving it alone generally are:
1. It's a pain
2. Copyright can't be reasserted unless meaningful changes were actually made
2a. Lawyers themselves disagree about the previous point
3. Updating the copyright year extends the legal horizon more than a century from now, which seems unlikely to be consequential to Deis